src/classes/services/ContactService.cls

```diff
@@ -0,0 +1,48 @@
+/**
+ * Created by Mariya on 14.11.2022.
+ */
+
+public with sharing class ContactService {
+    public static void createTeamMember(List<Contact> newContacts){
+
+        List<Contact> contactList = new List<Contact>();
+
+        List<Boutique_Resource__c> boutiqueList = new List<Boutique_Resource__c>();
+
+        for (Contact item: newContacts){
+            if (item.AccountId != null) {
+                contactList.add(item);
+            }
+        }
+
+        for (Contact item: contactList){
+            Boutique_Resource__c newTeamMember = new Boutique_Resource__c(Boutique__c = item.AccountId, Contact__c = item.Id, Name = item.LastName);
+            newTeamMember.RecordTypeId = '0127S000000c4WtQAI';
```

Comment:
CRITICAL
Do not hardcode Ids, as Ids can vary between environments. 
E.g. after deployment, this code could fail in PROD as RecordTypeId could be different.
Use dynamic Apex to get RT: 
`public static final Map<String, Schema.RecordTypeInfo> BOUTIQUE_RESOURCES_RTS_BY_DEV_NAMES = Schema.SObjectType.Boutique_Resource__c.getRecordTypeInfosByDeveloperName();`

And then get the needed Id by DeveloperName.
Think about Utility class for this and potentially getter or method for every RecType.

Reply:
I dont fully understand how correctly implement it.  I should create a method in ContactUtility class, that will return record type Id?  Something like this: 
    public static Map<String, Schema.RecordTypeInfo> getRecordTypeByName (){

        Map<String, Schema.RecordTypeInfo> BOUTIQUE_RESOURCES_RTS_BY_DEV_NAMES = Schema.SObjectType.Boutique_Resource__c.getRecordTypeInfosByDeveloperName();

        return BOUTIQUE_RESOURCES_RTS_BY_DEV_NAMES;
    }

Reply:
or just  

Id devRecordTypeId = Schema.SObjectType.Boutique_Resource__c.getRecordTypeInfosByName().get('Team Member').getRecordTypeId();